### ISSUE TITLE
Add links issue

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#4289]
When opening the Add Links dialogue I got the following message:

Error running R command(s)
Error in self$get_data_objects(data_name) : RwandaAll3 not found
The error occurred in attempting to run the following R command(s):

.temp_value<-InstatDataObject$get_keys(data_name="RwandaAll3")
OK   

I had earlier used this dialogue and set up a link.  But that data set was now closed.  Perhaps that is the problem!

**Comment by dannyparsons** [comment]
Yes I think an error because the previous data frame no longer exists. We should change the code to not display the error message in this case.

**Comment by lilyclements** [comment]
This is still a problem, and it crashes it altogether now! 

- Read in tricots data
- Go to "Add Links" dialog
- Close dialog
- Delete data frame 
- Reopen "Add Links" dialog
- Crashes and shuts down.
